modules/developer_manual/pages/app/fundamentals/info.adoc

```diff
@@ -302,7 +289,7 @@ IMPORTANT: This will be mandatory from version 11 onwards.
 The following sections are listed just for reference and should not be
 used because:
 
-* *public/remote*: Use api instead because you’ll have to use xref:core/externalapi.adoc[the external API], which is known to be buggy. It only works properly with GET/POST requests.
+* *public/remote*: Use api instead because you'll have to use xref:core/externalapi.adoc[the external API], which is known to be buggy. It only works properly with GET/POST requests.
```

Comment:
hmmm, this sentence confuses me a bit...
for me it reads the following: use the api but it is know to be buggy ??

Reply:
As this is separate to this PR, can you put this in a new issue, pls?